docs/reference/installation.rst

```diff
@@ -4,7 +4,7 @@ Installation
 Prerequisites
 -------------
 
-PHP 7.1 and Symfony >=3.4 or >= 4.2 are needed to make this bundle work, there are
+PHP 7.1 and Symfony >=4.3 are needed to make this bundle work, there are
```

Comment:
Not directly related, but note that this project isn't being built against PHP versions previous to 7.2:
https://github.com/sonata-project/SonataPageBundle/blob/c2cc2e7b70cd41547d96fc52778d950905d6d789/.travis.yml#L12-L16

Reply:
I'll change it in another PR that I'm working on to fix https://github.com/sonata-project/SonataPageBundle/issues/1134